tests/test_disp.py

```diff
@@ -151,32 +159,37 @@ def test_atm_single(dtype: torch.dtype, name: str) -> None:
 @pytest.mark.parametrize("dtype", [torch.float, torch.double])
 @pytest.mark.parametrize("name1", ["SiH4", "MB16_43_01"])
 @pytest.mark.parametrize("name2", ["SiH4"])
-def test_atm_batch(dtype: torch.dtype, name1: str, name2: str) -> None:
+@pytest.mark.parametrize("name3", ["AmF3"])
```

Comment:
Please just add AmF3 to the list of `name1`. It suffices to have two systems in the batch.